### DISCUSSION TITLE
What journal for PPG II?

### DISCUSSION THREAD

**Discussion by Meza-Torres** [pteridogroup/ppg#90] [General]
Dear all,
I was reading the IBC Workshop Summary, and I saw that the publication of PPG 2 is a topic of discussion. I read Taxon journal is a possibility. However, I would like to give my opinion.
I think that PPG 2 should be published in a journal of pteridology, as American Fern Journal or The Fern Gazette, or another journal of a pteridology society.
Amer Fern J. is a US journal, but I think it is the journal that best represents to community of pteridologists. While Fern Gazette is an excellent journal, Amer. Fern J. is a little better known in libraries, due to its wider distribution in the days of print publications (At least in the LIL and CTES libraries in Argentina).
Unfortunately, due to evaluations based on the impact factor of journals, supervisors currently advise against to students publishing in journals that are positioned in the q3 and q4 quartiles of Scimago, or low impact index.
The PPGI has about 1300 citations, which helped positioned the Journal of Systematics and Evolution very well.  PPG2 will undoubtedly have the same or more citations, which will also help position the journal that publishes it. This would be a good reason for PPG 2 to be published in a pteridology journal, to benefit its IF.  This will encourage students to submit their work to our pteridology journals without fear of academic evaluations.
For all these reasons, if I have to give my opinion, PPG 2 should be published in the American Fern Journal, or the Fern Gazette.
Best regard

**Comment by schwartsburd** [comment]
Hello there. I partially agree with Mezza-Torres - a pteridophyte journal would be a good a idea. On the other hand, I believe we have the most important and up to date classification and information on pteridophytes around the World. So, why not kicking as high as possible? Why not aiming to Nature or Science???!!! Impact factor is about 30.0 in these journals...

**Comment by GuguYanez** [comment]
I have mixed feelings about this. I agree with Esteban on the importance of publishing in journals that we, as a community, seek to strengthen and elevate. However, in this particular case, regardless of impact factor, I believe we should prioritize a journal with broad visibility within the botanical community.
Ultimately, what we all want is for the new classification to reach as wide an audience as possible—both within and beyond the scientific community (especially beyond!)—so that it can be adopted and used promptly. For this reason, it may be preferable to submit to a traditionally well-established botany journal, such as Taxon, or even the American Journal of Botany or the Botanical Journal of the Linnean Society (the latter having published APG IV).

**Reply by BlancaRFerns** [reply]
I agree Gugu in prioritizing broad visibility. Recently, I suggested the following journals:
Taxon (since it provides a broad audience of practicing taxonomists)
Systematic Botany (a classification might be of interest to editors)
Others: Botanical Journal of the Linnean Society, since APG IV was  published there.
PhytoKeys, because the report of the Nomenclature session will be published there.

**Comment by DrAshleyField** [comment]
Hi everyone, My preference is for a journal that is: (1) well circulated, (2) impactful, (3) published by an international society (rather than a national or regional society), and (4) is not limited in scope to ferns. The fern community already knows about PPG, it is the non-fern community that I would like to add to the audience. Of those on the ballot, _Taxon_ fits these criteria. It would be a pleasure if _Nature_ would take a classification publication. Ash

**Comment by joelnitta** [comment]
Thanks to all for contributing to the discussion and for voting on a journal.

_Taxon_ received a majority of votes, so that is where we will submit.

<img width="2196" height="924" alt="ppg2_journal_results" src="https://github.com/user-attachments/assets/b66b982a-2dd8-4142-874a-69f06c7e5a6f" />
